### DISCUSSION TITLE
Parameterize Normal Distribution with Precision Matrix using LinearOperator

### DISCUSSION THREAD

**Discussion by alexpeters1208** [cornellius-gp/gpytorch#2110] [Q&A]
Hello,

A few months ago, I posted a question about parameterizing the MVN in GPyTorch with a precision matrix rather than a covariance matrix. I put down this project for a few months and am picking it back up. Geoff's solution was as follows:

"The way to implement this in GPyTorch would be to implement a SparsePrecisionLazyTensor, which would be parameterized by a sparse precision matrix. You would then want to override the inv_matmul, inv_quad, and inv_quad_logdet to essentially perform matrix multiplication with the precision matrix."

Now that the LazyTensor functionality has been moved into the LinearOperator repo, would this advice still hold? Just to extend those 3 methods in the linear operator class to work with precision matrices? Thank you!

**Comment by gpleiss** [comment]
Hi @alexpeters1208 , this advice still holds! The `LinearOperator` migration should only introduce a few naming changes. In particular, `inv_matmul` has been renamed to `solve`.
